radius.go

```diff
@@ -27,46 +27,61 @@ func start_radius_server(db *sql.DB) {
 	// from the user is not found, send reject code 
 	// accept code instead otherwise
 	handler := func(w radius.ResponseWriter, r *radius.Request) {
+		log.Printf("Request")
 		// grabbing information
 		// note that we assume username = imsi
 		// optional ip is undefined
 		imsi := rfc2865.UserName_GetString(r.Packet)
 		// query through database to find if this imsi matches some IP
 		// if it does, write back the IP
 		// Reject otherwise
+		// QueryString := "SELECT ip FROM static_ips WHERE imsi = " + imsi
 		rows, err := db.Query("SELECT ip FROM static_ips WHERE imsi = ?", imsi)
 		defer rows.Close()
 		var res *radius.Packet
 		if err != nil {
-			// we should reject the user 
-			log.WithError(err).Error("Cannot find this customer in our db")
-			res = radius.New(radius.CodeAccessReject, []byte(`res`)) // what should we set the secret?
-		} else {
+			// sth wrong with our database 
+			log.WithError(err).Error("We cannot query through our databse")
+			// what should we behave here? reject the user? or should we just return?
```

Comment:
At the point this is a PR, we should have these issues resolved or start a discussion to figure it out.